py/gc.c

```diff
@@ -383,6 +383,10 @@ void gc_collect_start(void) {
     #endif
 }
 
+void gc_collect_ptr(void *ptr) {
+    gc_mark(ptr);
```

Comment:
Why not just rename gc_mark?

Reply:
I thought of calling gc_mark directly, but thought someday we might want `gc_collect_ptr()` to do something else before it calls `gc_mark()`. Having a routine called `gc_mark()` is really clear about what it does. It marks and only marks.

Reply:
okie